56
Loading data...
Load events...
../../Output/raw_data_with_events_to_python_all_comparisons_patient_479_sentences_blocks_135.mat
Generating MNE raw object...
Creating RawArray with float64 data, n_channels=1, n_times=126369280
    Range : 0 ... 126369279 =      0.000 ...  4212.309 secs
Ready.
0
Epoching data...
532 matching events found
0 projection items activated
Loading data for 532 events and 180001 original time points ...
0 bad epochs dropped
<Epochs  |  n_events : 532 (all good), tmin : -3.0 (s), tmax : 3.0 (s), baseline : None, ~730.6 MB, data loaded,
 'FIRST_WORD_TIMES_block_1': 152, 'FIRST_WORD_TIMES_block_3': 152, 'FIRST_WORD_TIMES_block_5': 152, 'KEY_PRESS_l_TIMES_block_1': 20, 'KEY_PRESS_l_TIMES_block_3': 28, 'KEY_PRESS_l_TIMES_block_5': 28>
('Original sampling rate:', 30000.0, 'Hz')
Traceback (most recent call last):
  File "main_epoch_macro_with_time_freq.py", line 218, in <module>
    epochs_resampled = epochs.copy().resample(params.downsampling_sfreq, npad='auto')
  File "/home/yl254115/.local/lib/python2.7/site-packages/mne/epochs.py", line 1445, in copy
    new = deepcopy(self)
  File "/volatile/anaconda2/lib/python2.7/copy.py", line 190, in deepcopy
    y = _reconstruct(x, rv, 1, memo)
  File "/volatile/anaconda2/lib/python2.7/copy.py", line 334, in _reconstruct
    state = deepcopy(state, memo)
  File "/volatile/anaconda2/lib/python2.7/copy.py", line 163, in deepcopy
    y = copier(x, memo)
  File "/volatile/anaconda2/lib/python2.7/copy.py", line 257, in _deepcopy_dict
    y[deepcopy(key, memo)] = deepcopy(value, memo)
  File "/volatile/anaconda2/lib/python2.7/copy.py", line 174, in deepcopy
    y = copier(memo)
MemoryError
